io.github.swanux.hbud.yml

```diff
@@ -0,0 +1,139 @@
+app-id: io.github.swanux.hbud
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: io.github.swanux.hbud
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --share=ipc
+  - --device=all
+  - --filesystem=xdg-run/gvfs
+
+cleanup:
+  - "/lib/cmake"
+  - "*.la"
+  - "*.a"
+  - "/share/doc"
+  - "/share/gtk-doc"
+  - "/share/locale"
+  - "/share/runtime"
+  - "/share/kde4"
+  - "/share/metainfo"
+  - "/share/vlc"
+  - "/share/icons/hicolor/16x16"
+  - "/share/icons/hicolor/32x32"
+  - "/share/icons/hicolor/48x48"
+  - "/share/icons/hicolor/256x256"
```

Comment:
why are you removing icons?

Reply:
To free up some space - as I'm only using libVLC as backend. I don't know if there's a proper build flag to omit icons in libVLC.